pkgs/by-name/co/corepack/package.nix

```
@@ -0,0 +1,79 @@
+{
+  lib,
+  stdenvNoCC,
+  yarn-berry,
+  nodejs,
+  fetchFromGitHub,
+  nix-update-script,
+  versionCheckHook,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "corepack";
+  version = "0.34.1";
+
+  src = fetchFromGitHub {
+    owner = "nodejs";
+    repo = "corepack";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-dZ5gVcrBNeyyZ10rU0HQaHUisL3sBZanTusPjMME8lk=";
+  };
+
+  patches = [
+    ./sqlite.patch
+  ];
+
+  nativeBuildInputs = [
+    nodejs
+    yarn-berry.yarnBerryConfigHook
+  ];
+
+  missingHashes = ./missing-hashes.json;
+  offlineCache = yarn-berry.fetchYarnBerryDeps {
+    inherit (finalAttrs) src missingHashes patches;
+    hash = "sha256-uboftr6iRQ3QSDIi3nf/p/hsjDXJxJ7DO/gp/8qYFGk=";
+  };
+
+  buildPhase = ''
+    runHook preBuild
+
+    ${lib.getExe yarn-berry} build
```

```suggestion
    yarn-berry build
```
add it to nativeBuildInputs